modules/ROOT/pages/download/nb26/index.adoc

```diff
@@ -0,0 +1,138 @@
+ (link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip.sha512[SHA-512],
+ link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip.asc[PGP ASC])
+
+*Source:*
+
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip?action=download[netbeans-{netbeans-version}-source.zip] 
+(link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.sha512[SHA-512], 
+link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.asc[PGP ASC])
+
+TIP: You can link:https://www.apache.org/dyn/closer.lua#verify[verify the integrity] of the downloaded files using the PGP signatures (.asc files) or hash (.sha512 files).
+The PGP keys used to sign this release are available link:https://downloads.apache.org/netbeans/KEYS[here].
+
+*Release Notes:*
+
+* link:https://github.com/apache/netbeans/releases/tag/{netbeans-version}[Github Link]
+
+== Installers and Packages
+
+- link:https://installers.friendsofapachenetbeans.org/[Friends of Apache NetBeans] - Windows, macOS and
+Linux (.deb / .rpm) built with link:https://github.com/apache/netbeans-nbpackage/[NBPackage].
+All include a local JDK runtime for the IDE to run on, for a self-contained out-of-the-box experience.
+- link:https://snapcraft.io/netbeans[Linux snap package]
+
+IMPORTANT: These installers and packages are provided by various NetBeans committers as a convenience.
+While built using the Apache NetBeans release, they are not releases of the Apache Software Foundation.
+They may include other contents (eg. JDK) under additional license terms.
+
+== Deployment Platforms
+
+Apache NetBeans {netbeans-version} supports running on JDK 24, 21 or 17.
+
+TIP: The Runtime JDK NetBeans uses does not influence the JDK range projects can use.
+
+TIP: Use the latest JDK update release for the best experience (see link:https://javaalmanac.io/jdk/[Java Version Almanac]).
+
+== Known Issues
+
+* NetBeans 23+ does no longer support String Templates, link:https://github.com/apache/netbeans/discussions/7620[more info]
+
+* On windows you may encounter the following JDK issue: "Comparison method violates its general contract"
++
+resolution: use latest JDK update release, link:https://github.com/apache/netbeans/discussions/6361[more info]
+
+* Windows/ARM is currently not a fully supported platform
+
+* link:https://github.com/apache/netbeans/issues[All Issues on GitHub]
+
+* link:https://netbeans.apache.org/participate/report-issue.html[How to Report an Issue]
+
+== Building from Source
+
+1. Unzip link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip[netbeans-{netbeans-version}-source.zip]
+in a directory of your liking.
+
+[start=2]
+. Follow the build instructions in the README
```

Comment:
to simplify it should be ok to have only . on first block (removing 1), remove white line + [start=2] and number should follow.

```
. Unzip link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip[netbeans-{netbeans-version}-source.zip]
in a directory of your liking.
. Follow the build instructions in the README
```